understanding/20/name-role-value.html

```diff
@@ -14,33 +14,40 @@ <h2>Intent of Name, Role, Value</h2>
       
       
       <p>The intent of this Success Criterion is to ensure that Assistive Technologies (AT)
-         can gather information about, activate (or set) and keep up to date on the status of
+         can gather appropriate information about, activate (or set) and keep up to date on the status of
          user interface controls in the content.
       </p>
       
       <p>When standard controls from accessible technologies are used, this process is straightforward.
          If the user interface elements are used according to specification the conditions
-         of this provision will be met. (See examples  of Success Criterion 4.1.2 below)
+         of this provision will be met. (See examples of Success Criterion 4.1.2 below)
       </p>
       
       <p>If custom controls are created, however, or interface elements are programmed (in
          code or script) to have a different role and/or function than usual, then additional
-         measures need to be taken to ensure that the controls provide important information
+         measures need to be taken to ensure that the controls provide important and appropriate information
          to assistive technologies and allow themselves to be controlled by assistive technologies.
       </p>
+     
+      <p>What roles and states are appropriate to convey to assistive technology will be determined
+         by what the control represents. Specifics about such information are defined by other
```

Comment:
```suggestion
         on what the control represents. Specifics about such information are defined by other
```

(goes hand in hand with the previous suggestion)